articles/azure-large-instances/workloads/epic/lun-and-wwid-mapping.md

```diff
@@ -0,0 +1,45 @@
+---
+title: LUN and WWID mapping
+description: Explains ALI LUN and WWID mapping.
+ms.topic: conceptual
+author: jjaygbay1
+ms.author: jacobjaygbay
+ms.service: baremetal-infrastructure
+ms.date: 06/01/2023
+---
+
+# LUN and WWID mapping
+
+|LUN                     |WWID                    |
+|------------------------|------------------------|
+|t050_rhel_epic_db_lun_1 |383147453724544676712d69|
+|t050_rhel_epic_db_lun_9 |383147453724544676712d6d|
+|t050_rhel_epic_db_lun_10 |383147452d2454546b2d4237|
+|t050_rhel_epic_db_lun_2 |383147452d2454546b2d4233|
+|t050_rhel_epic_db_lun_11 |383147453724544676712d6e|
+|t050_rhel_epic_db_lun_3 |383147453724544676712d6a|
+|t050_rhel_epic_db_lun_12 |383147452d2454546b2d4238|
+|t050_rhel_epic_db_lun_4 |383147452d2454546b2d4234|
+|t050_rhel_epic_db_lun_13 |383147453724544676712d6f|
+|t050_rhel_epic_db_lun_5 |383147453724544676712d6b|
+|t050_rhel_epic_db_lun_14 |383147452d2454546b2d4239|
+|t050_rhel_epic_db_lun_6 |383147452d2454546b2d4235|
+|t050_rhel_epic_db_lun_15 |383147453724544676712d70|
+|t050_rhel_epic_db_lun_7 |383147453724544676712d6c|
+|t050_rhel_epic_db_lun_16 |383147452d2454546b2d422d|
+|t050_rhel_epic_db_lun_8 |383147452d2454546b2d4236|
+|t050_rhel_epic_inst_lun1 |383147453724544676712d73|
+|t050_rhel_epic_inst_lun2 |383147452d2454546b2d4243|
+|t050_rhel_epic_jrn_lun1 |383147453724544676712d71|
+|t050_rhel_epic_jrn_lun3 |383147453724544676712d72|
+|t050_rhel_epic_jrn_lun2 |383147452d2454546b2d4241|
+|t050_rhel_epic_jrn_lun4 |383147452d2454546b2d4242|
```

Comment:
@vasthomsft These LUNs and WWIDs are not from test environment, should we show some dummy values here?

Reply:
We should remove this section altogether. This stands out very isolated without specific guidance to end users. @ryellajo12 , @makotrik , if you folks agree, let's remove this section.

Reply:
@vasthomsft have you received agreement to remove this section?

Reply:
@vasthomsft yes, please remove

Reply:
@makotrik lines 33-36 have been removed - issue is resolved.